aws/configs/amazon-cloudwatch-agent.json

```diff
@@ -92,43 +43,40 @@
             }
         },
         "append_dimensions": {
-            "ImageId": "${aws:ImageId}",
             "InstanceId": "${aws:InstanceId}",
-            "InstanceType": "${aws:InstanceType}",
             "AutoScalingGroupName": "${aws:AutoScalingGroupName}"
         },
         "aggregation_dimensions": [
-            ["ImageId"],
-            ["InstanceId", "InstanceType"],
             []
-        ],
-        "force_flush_interval": 30
+        ]
     },
     "logs": {
         "logs_collected": {
             "files": {
                 "collect_list": [
                     {
                         "file_path": "/var/log/spacelift/error.log",
-                        "log_group_name": "spacelift-errors.log",
+                        "log_group_name": "/spacelift-worker-image/spacelift-errors.log",
                         "log_stream_name": "{instance_id}-{ip_address}",
-                        "timezone": "UTC"
+                        "timezone": "UTC",
+                        "retention_in_days": 7
                     },
                     {
                         "file_path": "/var/log/spacelift/info.log",
-                        "log_group_name": "spacelift-info.log",
+                        "log_group_name": "/spacelift-worker-image/spacelift-info.log",
                         "log_stream_name": "{instance_id}-{ip_address}",
-                        "timezone": "UTC"
+                        "timezone": "UTC",
+                        "retention_in_days": 7
                     },
                     {
                         "file_path": "/opt/aws/amazon-cloudwatch-agent/logs/amazon-cloudwatch-agent.log",
-                        "log_group_name": "amazon-cloudwatch-agent.log",
+                        "log_group_name": "/spacelift-worker-image/amazon-cloudwatch-agent.log",
                         "log_stream_name": "{instance_id}-{ip_address}",
-                        "timezone": "UTC"
+                        "timezone": "UTC",
+                        "retention_in_days": 7
                     }
                 ]
             }
-        },
-        "force_flush_interval" : 15
```

Comment:
Just wanna double check this one - if I'm reading the docs this will reduce the flush interval to the default value of 5 seconds. That makes sense to me, but just wanna double check the reasoning (e.g. is it to make it less likely logs are lost, etc?).

Reply:
Good catch, I assumed the default interval was higher! The reasoning was to move this file as close to default configuration as possible. Since log ingest has no per-API-call charge, I think it's better overall to stick with the default value here, but I'll change it if you prefer. I don't think 5 vs 15 seconds of data loss in a crash is meaningful.

Reply:
I think using the default is fine.